BuildGenerated.sh

```diff
@@ -124,7 +124,7 @@ if [ -z ${SKIPGENERATE+x} ]; then
         ;;
       *)
         echo Generating: \'$name\'
-        run_csharp_generator "$jsonfile" "$CODE_GENERATION_DIR" features.json
+        run_csharp_generator "$jsonfile" "$CODE_GENERATION_DIR" features.json "EnumStorage/$name.json"
```

Comment:
Just to make super sure, the EnumStorage folder is not deleted anywhere, rigth?
I think the EnumStorage folder is a sibling folder of $CODE_GENERATION_DIR right?

Reply:
Yes, that's right - it's top-level.